### PR TITLE
feat: added seo meta tegs

### PR DESCRIPTION
**Pull request by Arfey**

### CODE REVIEW

File: create_aio_app/template/{{cookiecutter.project_name}}/docs/source/index.rst [hunk 1]
```diff
@@ -55,3 +55,10 @@ Indices and tables
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+.. This part only for the main site https://create-aio-app.readthedocs.io/
```
Comment (webknjaz) [inline create_aio_app/template/{{cookiecutter.project_name}}/docs/source/index.rst]: Better do this through epilogue replacement...

Comment (Arfey) [reply create_aio_app/template/{{cookiecutter.project_name}}/docs/source/index.rst]: are u talking about that?
```
.. |reST| replace:: reStructuredText

Yes, |reST| is a long word, so I can't blame anyone for wanting to
abbreviate it.

```

Comment (webknjaz) [reply create_aio_app/template/{{cookiecutter.project_name}}/docs/source/index.rst]: Yep. This way add whatever you want only for our RTD but don't ship it to users in templates.

Example: https://github.com/cherrypy/cherrypy/blob/master/docs/conf.py#L66-L70

Comment (Arfey) [reply create_aio_app/template/{{cookiecutter.project_name}}/docs/source/index.rst]: thx

**Pull request merged**